strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/io/ArtifactOutputStreamTest.java

```diff
@@ -42,13 +42,22 @@
 {
 
     public static final String REPOSITORY_RELEASES = "aos-releases";
+    
+    private static boolean initialized;
```

Comment:
> I think that we should be able to perform `@PostСonstruct` methods before each test. 

Of course, I agree. But you need to know what you're doing and initialize some things once, if they are shared between tests, or clean it up properly. This "workaround" was a solution for incorrect resource locking attempt under Windows as the exception stated:

> org.apache.lucene.store.LockObtainFailedException: Lock obtain timed out: NativeFSLock@D:\workspace_neon\workspace_strongbox\strongbox\strongbox-storage\strongbox-storage-layout-providers\strongbox-storage-maven-layout-provider\target\strongbox-vault\storages\storage0\m2lp-releases\.index\local\write.lock: java.nio.channels.OverlappingFileLockException

So, in my opinion, this concrete exception said that you're handling resources incorrectly. So either clean it up properly after every test case, or initialize it once. This "workaround" was a second option.

> we need to fix the root of the problem, wherever it is.

The root problem was an issue with resources handling (at least under Windows OS). Exception above showed up that file locking policy is more restrictive under Windows than Linux.